pyproject.toml

```diff
@@ -29,10 +29,3 @@ urls = { Homepage = "https://github.com/tpgillam/mt2" }
 requires = ["setuptools>=61.0", "numpy"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools.packages.find]
-where = ["src"]
-include = ["mt2*"]
-namespaces = false
```

Comment:
Default setuptools behaviour picks up all the packages now it seems.

Reply:
The default behavior also autodetects things that look like namespaces. I was restricting it to mt2 prefixed things and removing the auto-namespace detection given some of the peculiarities of the package thus far.

Reply:
`_mt2` wouldn't match this pattern though, I think? My hunch was that we might have been additionally including `mt2*` but not actually excluding other stuff... but I wasn't too sure. Hence I figured the default wouldn't hurt.